dev/com.ibm.ws.config/resources/com/ibm/ws/config/internal/resources/ConfigMessages.nlsprops

```diff
@@ -451,5 +451,9 @@ error.extendsAlias.collision=CWWKG0100E: The {2} ibm:extendsAlias attribute is d
 error.extendsAlias.collision.explanation=The ibm:extendsAlias attribute must be unique within a single extends hierarchy.
 error.extendsAlias.collision.useraction=Make each ibm:extendsAlias attribute unique within the extends hierarchy.
 
+warning.binding.config=CWWKG0101W: The configuration with the persisted identity (PID) {0} will be bound to the bundle {1}. The configuration will not be visible to other bundles.  
+warning.binding.config.explanation=A configuration is bound to the calling bundle when the getConfiguration method of the ConfigurationAdmin interface is called for the first time. Once the configuration is bound, it is no longer visible to other bundles.  
```

Comment:
What is doing the actions in the first sentence? If we could restructure the sentence to mention what's doing the actions, that might make the sentence clearer. For example...
"A <subject> binds a configuration to the calling bundle when a <subject> calls the getConfiguration method of the ConfigurationAdmin interface for the first time."

Reply:
It could be anything that can call the ConfigurationAdmin API, so there isn't really a single subject that we could put in the message.